numpy/f2py/__init__.py

```diff
@@ -35,8 +35,11 @@ def compile(source,
         additional parameters passed to f2py
     verbose: bool, optional
         print f2py output to screen
+    source_fn : str, optional
+        filename where the fortran source is dumped
```

Comment:
Could use a bit more explanation. I can't figure out what "dumped" means in this context ;)